src/Assets/TestProjects/BlazorHosted/blazorwasm/App.razor

```diff
@@ -0,0 +1,8 @@
+﻿<Router AppAssembly="@typeof(Program).Assembly" PreferExactMatches="true">
+    <Found Context="routeData">
```

Comment:
And be sure to add individual test project assets to the code owner too. These tests assets are not trivial

Reply:
Updated codeowners with all the newly added directories.